### ISSUE TITLE
Sway description update

### ISSUE THREAD

**Issue by Bubble-one** [GDevelopApp/GDevelop-extensions#622]
### Description

Updated description to fix link to gif and to remove an alert warning of 'for each' slow downs now that Bouh removed that part.

Also put the updated extension with removed 'for each' and corrected description in the game example.

### Checklist

- [X] I've followed all of [the best practices](http://wiki.compilgames.net/doku.php/gdevelop5/extensions/best-practices).
- [X] I confirm that this extension can be integrated to this GitHub repository, distributed and MIT licensed.
- [X] I am aware that the extension may be updated by anyone, and do not need my explicit consent to do so.

### What tier of review do you aim for your extension?

Reviewed

### Example file

[SwayExample.zip](https://github.com/GDevelopApp/GDevelop-extensions/files/9712444/SwayExample.zip)

### Extension file

[Sway.json.zip](https://github.com/GDevelopApp/GDevelop-extensions/files/9712313/Sway.json.zip)

**Comment by Bubble-one** [comment]
Hi, sorry, I hope this is the correct way to suggest an update and hope the bot doesn't keep confusing things.

Thank you everybody for your work and thoughts on my extension.

Thank you @Bouh for fixing my mistakes. 

Bouh, now that you've removed the unncessary 'for each' condition, I've updated the description to remove this line:
This behavior uses *For each object* events so be aware of slow downs that may occur from many instances of Sway objects.

I've also fixed my incorrect formatting of the link to the example gif.

It used to say this:
![Imgur](https://i.imgur.com/dOyTgbN.gifv)

Now it says this:
Link to a gif showing an example of Sway in use:
[Imgur](https://i.imgur.com/dOyTgbN.gifv)

Everything else is the same.

Thank you.
